src/components/molecules/Inbox.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <h1 class="font-heading font-bold text-4xl pl-3 pt-4">Inbox</h1>
+  <h1 class="font-heading font-bold text-4xl pl-3 py-4">Inbox</h1>
```

Comment:
In the design the word Inbox sits lower in the div. Almost resting on the bottom of it

![image](https://user-images.githubusercontent.com/69208837/123100699-54575f00-d401-11eb-8c51-f5c449b342c0.png)

Reply:
Added more padding to the top and less to the bottom. Let me know if there's anything else that you've noticed